lib/logger.js

```diff
@@ -99,6 +99,16 @@ function Logger(key, options) {
         throw new Error('Invalid URL');
     }
 
+    if (options.flushLimit && Number.isInteger(options.flushLimit)) {
+        this._flushLimit = options.flushLimit;
+    } else {
+        this._flushLimit = configs.FLUSH_BYTE_LIMIT;
+    }
+
+    if (options.flushInterval && Number.isInteger(options.flushInterval)) {
+        this._flushInterval = options.flushInterval;
```

Comment:
Set `this._flushInterval` in the same way you set `this._flushLimit`

Reply:
I can do that, but the logic for this property is different.